tests/robustness/main_test.go

```diff
@@ -83,15 +87,40 @@ func TestRobustnessRegression(t *testing.T) {
 	}
 }
 
+func testResultsDirectory(t *testing.T) string {
```

Comment:
Move this function to the bottom of file. It's much less important function, so it should be lower. Code should be read from top to the bottom. We shouldn't put low importantance function on the top.